frontend/lib/src/components/shared/BaseColorPicker/BaseColorPicker.tsx

```diff
@@ -142,7 +139,9 @@ class BaseColorPicker extends React.PureComponent<
             </StyledChromePicker>
           )}
         >
-          <StyledColorPreview style={previewStyle}>
+          <StyledColorPreview
+            style={{ cursor, pointerEvents: disabled ? "none" : "auto" }}
```

Comment:
nit: Any reason this shouldn't be in the styled component itself? Seems like cursor depends on disabled, so keeping it simple like

`<StyledColorPreview isDisabled={disabled}>` and handling `cursor` and `pointerEvents` in there would clean things up.

Reply:
Yep 👍  I refactored the color picker and moved all the styles into the styled componnt